packages/firestore/test/integration/bootstrap.ts

```diff
@@ -25,7 +25,6 @@ import '../../index';
  */
 
 // 'context()' definition requires additional dependency on webpack-env package.
-// tslint:disable-next-line:no-any
```

Comment:
Why are we disabling no-explicit-any for tests?  I think since Firestore is already `any`-clean (other than suppressions like this), we'd like to stay that way...

It looks like we also have a `"@typescript-eslint/tslint/config": "off` override...  why?  In particular, I think that means our [checks](https://github.com/firebase/firebase-js-sdk/blob/84725bce4d908aeb8bb12fd2523718e383f32dca/config/.eslintrc.json#L144) to prevent accidentally checking in `it.only()` tests etc. will be disabled?

Reply:
Sure, we can enable it just for Firestore if other teams find it useful.  We do very little mocking (I'm not a big fan), so we don't have to do a lot of `any` casting...

Reply:
Done!